source/agora/api/Validator.d

```diff
@@ -120,4 +120,22 @@ public interface API : agora.api.FullNode.API
     ***************************************************************************/
 
     public void postBlockSignature (ValidatorBlockSig block_sig);
+
+    /// Interfaces that are only enabled in debug builds
+    debug
```

Comment:
```suggestion
    debug (AgoraDebugAPIs)
```

Then use with `"debugVersions": [ "AgoraDebugAPIs" ]` in `dub` or `-debug=AgoraDebugAPIs` manually.